helper-cli/src/main/kotlin/commands/classifications/ImportCommand.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands.classifications
+
+import com.fasterxml.jackson.module.kotlin.readValue
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.convert
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.file
+
+import java.net.URL
+
+import org.apache.logging.log4j.kotlin.Logging
+
+import org.ossreviewtoolkit.model.jsonMapper
+import org.ossreviewtoolkit.model.licenses.LicenseCategorization
+import org.ossreviewtoolkit.model.licenses.LicenseCategory
+import org.ossreviewtoolkit.model.licenses.LicenseClassifications
+import org.ossreviewtoolkit.model.yamlMapper
+import org.ossreviewtoolkit.utils.common.expandTilde
+import org.ossreviewtoolkit.utils.spdx.SpdxSingleLicenseExpression
+
+internal class ImportCommand : CliktCommand(
+    help = "Import license classifications from supported providers to ORT format."
+) {
+    private val provider by argument(
+        help = "The name of the provider to import license classifications from. Must be one of " +
+                "${enumValues<LicenseClassificationProvider>().map { it.name }}."
+    ).enum<LicenseClassificationProvider>()
+
+    // TODO: Make this default to the provider name with clikt 4, see https://github.com/ajalt/clikt/issues/381.
+    private val prefix by option(
+        "--prefix", "-p",
+        help = "A prefix to use for all category names declared by the provider's classifications."
+    )
+
+    private val licenseClassificationsFile by option(
+        "--output", "-o",
+        help = "The file to write the provided license classifications to."
+    ).convert { it.expandTilde() }
+        .file(mustExist = false, canBeFile = true, canBeDir = false, mustBeWritable = false, mustBeReadable = false)
+        .convert { it.absoluteFile.normalize() }
+
+    override fun run() {
+        val classifications = provider.getClassifications().run {
+            prefix?.let { prefixCategoryNames(it) } ?: this
+        }
+
+        val yaml = yamlMapper.writeValueAsString(classifications)
```

Comment:
Please sort the categories by name and the categorizations by id.

Reply:
I'd prefer to keep the discussion isolated, my original intention was that I believe 

1. It's more valuable to have the output sorted than having append or even random order

In your above comment there are the points

2. Should license classifications be changed to enforce sorted order
3. Should a wrapping / companion class be added
4. probably more

As 1. is pretty much independent of all the rest and can be implemented with typing just a couple of characters, I believe it's more appropriate to decide 1. in isolation an implement already now what we believe is best suited for the output.

Reply:
Ok, I've implemented the sorting.

Reply:
thx!